packages/python3/packaging

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e -x
+
+apt install -y libffi-dev
+tar xzf Python-3.7.4.tgz
+pushd Python-3.7.4
```

Comment:
Can this be changed to `Python-3.7-*.tgz` to be a bit more generic for patch releases?